_posts/2021-02-09-an-introduction-to-soliditys-fuzz-testing-approach.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+published: true
+title: 'An Introduction to Solidity's Fuzz Testing Approach'
+date: '2021-02-09'
+author: Bhargava Shastry
+category: Explainers
+---
+
+Security vulnerabilities and bugs detract from software quality.
+To discover them early, we have adopted fuzz testing: feeding randomly generated programs to the Solidity compiler and observing the compilation runtime and code generated.
+
+Since Q1 2019, the Solidity compiler is fuzz tested via Google's [open-source software fuzz][1] (oss-fuzz) framework.
```

Comment:
Does the link render properly with the space?

Reply:
ah, it's just a regular space, not part of the link, right?

Reply:
Yeah, the space is part of the hyperlinked text iiuc